src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    private static String command;
+    private static String description;
+    private static Task task;
+    private static boolean isTask;
+    private static boolean isCommand;
+    private static final TaskList list = new TaskList();
```

Comment:
Could this be renamed as tasksList? It might make it clearer it is to store multiple tasks